docs/source/cookbook/ccdc.rst

```diff
@@ -1,10 +1,475 @@
+
+.. thumbnail:: ../_images/cookbook/ccdc_asset/sensor_selection_planet.png
+    :title: Sensor Selection - Planet
+    :width: 49%
+    :align: center
+    :group: ccdc-asset-recipe
+
+|
+In both cases, the data already needs to reside within Earth Engine as an *ImageCollection asset*, whose ID needs to be filled in the respective field.
+
+In case you want to use the NICFI Level-1 basemaps, you can use the already existing assets within Earth Engine, given that you enabled the access feature as explained `here <https://developers.planet.com/docs/integrations/gee/nicfi/>`_. The NICFI Level-1 assets are splitted by continent and have the following Asset IDs:
+
+    - projects/planet-nicfi/assets/basemaps/africa
+    - projects/planet-nicfi/assets/basemaps/asia
+    - projects/planet-nicfi/assets/basemaps/americas
+
+
+.. tip::
+    For data ordered through the Planet API (i.e. daily imagery or custom basemaps other than NICFI Level 1 data), you can specify Earth Engine as the download location.
+
+Using CCDC with Planet has not been explored widely, so that again the optimal selection of the breakpoint bands depends on testing it out by yourself. However, in accordance with Landsat based analysis we suggest to use the Green , Red and NIR bands to get started.
+
+
+Pre-processing options
+^^^^^^^^^^^^^^^^^^^^^^
+
+
+Optical data
+"""""""""""""
+
+.. warning::
+
+    This section is optional as these parameters are set by default.
+
+    -   correction: :code:`None`
+    -   cloud detection: :guilabel:`QA bands`, :guilabel:`Cloud score`
+    -   cloud masking: :guilabel:`moderate`
+    -   snow masking: :guilabel:`on`
+
+Multiple pre-processing parameters can be set to improve the quality of the provided images. SEPAL has gathered 4 of them in the form of these interactive buttons. If you think others should be added to hesitate to mention it in our `issue tracker <https://github.com/openforis/sepal/issues/new/choose>`__.
+
+**Correction**
+
+-   :guilabel:`surface reflectance`: Use scenes atmospherically corrected surface reflectance.
+-   :guilabel:`BRDF correction`: Correct for bidirectional reflectance distribution function (BRDF) effects.
+
+**Cloud detection**
+
+-   :guilabel:`QA bands`: use pre-created QA bands from datasets
+-   :guilabel:`Cloud score`: use cloud scoring algorithm
+
+**Cloud masking**
+
+-   :guilabel:`Moderate`: rely only on image source QA bands for cloud masking
+-   :guilabel:`Aggressive`: rely on image source QA bands and a cloud scoring algorithm for cloud masking. This will probably mask out some built-up areas and other bright features.
+
+**Snow masking**
+
+-   :guilabel:`On`: mask snow. This tends to leave some pixels with shadowy snow
+-   :guilabel:`Off`: don't mask snow. Note that some clouds might get misclassified as snow, and because of this, disabling snow masking might lead to cloud artifacts.
+
+
+.. thumbnail:: ../_images/cookbook/time_series/pre_processing.png
```

Comment:
same comment as for the date selection, even though it's the same, I think it worth the extra screenshot